package.json

```diff
@@ -58,7 +58,7 @@
     "inquirer": "0.11.0",
     "lodash": "^4.5.1",
     "log-update": "^1.0.2",
-    "minimist": "^1.2.0",
+    "minimist": "git://github.com/mulesoft/minimist.git",
```

Comment:
For sure. This was required to make the tests pass. Will change this as soon as https://github.com/substack/minimist/pull/113 is merged.